### DISCUSSION TITLE
Menu in small screens

### DISCUSSION THREAD

**Discussion by ahmedabbous** [TDRA-ae/aegov-dls#25] [General]
The menu items on small screens are displayed on two rows which is being reported by many of our clients. Can you please share your feedback on this

![Screenshot 2024-03-05 100316](https://github.com/TDRA-ae/aegov-dls/assets/62495069/0d11532e-f5d6-4673-9bcd-3739b0a48fa8)

**Comment by moiz-conceptlz** [comment]
@ahmedabbous Can you please tell me what resolution this is ?

**Reply by ahmedabbous** [reply]
14 Inch screens  or (1522px width and less)
